src/Inciter/Transporter.h

```diff
@@ -195,7 +202,7 @@ class Transporter : public CBase_Transporter {
     void rowcomplete();
 
     //! Reduction target summing total mesh volume
-    void vol( tk::real v );
+    void totalvol( tk::real v );
```

Comment:
Maybe ``redn_''? I don't like prefixes but you did that elsewhere and it at least is indicative